README.md

```diff
@@ -43,27 +43,24 @@ ARGUMENTS
 ```
   Usage: 
 
-      easy-split-multi-deposit.sh [{--springfield-inbox|-s} <dir>] [{--staging-dir|-d} <dir>] <multi-deposit-dir> <output-deposits-dir> <datamanager>
+      easy-split-multi-deposit.sh [{--staging-dir|-s} <dir>] <multi-deposit-dir> <output-deposits-dir> <datamanager>
```

Comment:
generally agreed, but given that we do not yet use this tool widely (except for testing by Jeroen) I don't see this causes many problems.